common/achievement/metric.ts

```diff
@@ -22,9 +22,31 @@ function createMetric<T extends ActionID[], K extends ContextForActions<T>>(metr
 }
 
 const batchOfMetrics = [
-    createMetric('student_onboarding_registered', ['student_registration_started'], (context) => {
+    createMetric('onboarding_registered', ['requestedToken'], () => {
```

Comment:
this are also test metrics, so we should also remove them before merging. The metrics we need were added with another PR

Reply:
all of the metrics in the array or just the ones that do not exist anymore?